Bundle/WidgetBundle/Builder/WidgetFormBuilder.php

```diff
@@ -176,16 +177,21 @@ public function buildWidgetForm(Widget $widget, View $view, $businessEntityId =
         //are we updating or creating the widget?
         if ($widget->getId() === null) {
             $viewReference = $view->getReference();
-            $formUrl = $router->generate('victoire_core_widget_create',
-                [
-                    'mode'              => $formMode,
-                    'viewReference'     => $viewReference->getId(),
-                    'slot'              => $widget->getSlot(),
-                    'type'              => $widget->getType(),
-                    'businessEntityId'  => $businessEntityId,
-                    'positionReference' => $position,
-                ]
-            );
+            $params = [
+                'viewReference'     => $viewReference->getId(),
+                'slot'              => $slotId,
+                'type'              => $widget->getType(), // @todo: use the config
```

Comment:
The method getType() does nasty stuff to guess the widget name, I think this improvment should be done in an other PR